src/components/RenderComments.jsx

```diff
@@ -0,0 +1,24 @@
+function RenderComments({comment}) {
+
+    const dateFormat= new Date(comment.created_at)
```

Comment:
Good work formatting the date. Could this logic be extracted into another file for use as a util? It could more easily be used elsewhere that way and avoid clogging up our component. Make sure you test it too.